ee/hogai/graph/mixins.py

```diff
@@ -77,3 +79,36 @@ def project_timezone(self) -> str | None:
         Returns the timezone of the project, e.g. "PST" or "UTC".
         """
         return self._team.timezone_info.tzname(self._utc_now_datetime)
+
+    def _get_debug_props(self, config: RunnableConfig) -> dict[str, Any]:
```

Comment:
Question: Would it add value to also track assistant mode and message count in the state at the moment of exception? It surely can be tracked down the line, so curious if it would make sense to have them right away.